packages/tools/pie-css/test/scss/unit/settings/mq.spec.ts

```diff
@@ -0,0 +1,219 @@
+import {
+    describe,
+    it,
+    expect,
+} from 'vitest';
+
+import { compileCss, stripCSSWhitespace } from '../../../../utilities/compileCss';
+
+describe('settings :: mq', () => {
+    it('should render the expected breakpoint variable', () => {
+        // Arrange
+        const scssToTest = `
+          @use "sass:map";
+          @use 'settings' as *;
+
+          .foo {
+            @media (max-width: map.get($breakpoints, "sm")) {
+                color: red;
+            }
+          }
+        `;
+
+        const expectedCss = stripCSSWhitespace(`
+          @media (max-width:600px) {
+            .foo {
+              color: red;
+            }
+          }
+        `);
+
+        // Act
+        const css = stripCSSWhitespace(compileCss(scssToTest));
+
+        // Assert
+        expect(css).toBe(expectedCss);
+    });
+    it('should render the expected CSS media query', () => {
+        // Arrange
+        const scssToTest = `
+          @use 'settings' as *;
+
+          .foo {
+            @include media('<=sm') {
+                color: red;
+            }
+          }
+        `;
+
+        const expectedCss = stripCSSWhitespace(`
+          @media (max-width: 600px) {
+            .foo {
+              color: red;
+            }
+          }
+        `);
+
+        // Act
+        const css = stripCSSWhitespace(compileCss(scssToTest));
+
+        // Assert
+        expect(css).toBe(expectedCss);
+    });
+    it('should render the xs media query breakpoint shortcut', () => {
+        // Arrange
+        const scssToTest = `
+          @use 'settings' as *;
+
+          .foo {
+            @include media('>=xs') {
```

Comment:
The test names do not mention the use of `>=` - should that be included and do we want to test all variations?